src/CommunityToolkit.Maui.UnitTests/Converters/StateToBoolConverterTests.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System.ComponentModel;
 using System.Globalization;
 using CommunityToolkit.Maui.Converters;
+using CommunityToolkit.Maui.Core;
```

Comment:
Do we need this?

Reply:
Yup, because `public enum StateLayout` moved from `CommunityToolkit.Maui` to `CommunityToolkit.Maui.Core`